cdk/lib/recipes-backend.ts

```diff
@@ -84,11 +84,33 @@ export class RecipesBackend extends GuStack {
       description: "ARN of the SNS topic to use for data submissions"
     });
 
-		const faciaSNSTopicARNParam = new GuParameter(this, 'faciaSNSTopicParam', {
-			default: `/${this.stage}/${this.stack}/${app}/facia-sns-topic-arn`,
-			fromSSM: true,
-			description: 'The ARN of the facia-tool SNS topic that emits curation notifications',
-		});
+	const faciaSNSTopicARNParam = new GuParameter(this, 'faciaSNSTopicParam', {
+		default: `/${this.stage}/${this.stack}/${app}/facia-sns-topic-arn`,
+		fromSSM: true,
+		description: 'The ARN of the facia-tool SNS topic that emits curation notifications',
+	});
```

Comment:
It looks like the difference in this paragraph is just the number of tabs used for indent. This file seems to use tabs and spaces for indentation in different places, and I suspect this indent is not actually what we want: to match the paragraphs above (which use spaces), we should probably use the existing indentation here, with a tab-width of 2 spaces.

Maybe settling on tabs or spaces in the file would be good though.

Reply:
It's interesting I didn't spot that – I've turned on whitespace rendering in my IDE, which should make it much clearer when VSCode is making interesting indentation choices! Thanks for spotting, [d927337](https://github.com/guardian/recipes-backend/pull/51/commits/d9273377dd62ec2250e3fa0e75760b549c9222d0)

Reply:
Configuring an automatic formatter is probably the easiest way to get consistency without ever having to worry about it: are we using prettier here? (Something to look at in a subsequent PR rather than this one)

Reply:
Strong advocate of automatic linting, and have mentioned it in previous PR reviews on this repo, but shamefully never got around to implementing myself! This is a good cue to bump it up the priority list, agreed.